blueocean-dashboard/src/main/js/OrganisationPipelines.jsx

```diff
@@ -30,14 +30,12 @@ class OrganisationPipelines extends Component {
         return {
             pipelines,
             pipeline,
-            params,
-            location,
         };
     }
 
     componentWillMount() {
         if (this.context.config) {
-            this.props.fetchPipelinesIfNeeded(this.context.config);
+            this.props.fetchPipelinesIfNeeded(this.context.config, this.props.organization);
             const _this = this;
```

Comment:
It probably makes a lot more sense just to check the route params here and pass them in. Actually the more I explain the old code, the less I like it.